scripts/build/build-utils.ps1

```diff
@@ -147,6 +147,15 @@ function Invoke-UnitTests([string]$binPath, [bool]$failsIfNotTest) {
 
     & (Get-VsTestPath) $testFiles /Parallel /Enablecodecoverage /InIsolation /Logger:trx /TestAdapterPath:$testDirs
     Test-ExitCode "ERROR: Unit Tests execution FAILED."
+
+    $testProjFileName = "tests\SonarAnalyzer.UnitTest\SonarAnalyzer.UnitTest.csproj"
+    & dotnet build $testProjFileName
```

Comment:
Line 138 expects `*.UnitTest.dll` to exist already. Why is it build here then?

Reply:
I actually had to pass the buildConfiguration given to the script